Marlin/src/pins/stm32g0/pins_BTT_EBB42_V1_1.h

```diff
@@ -105,8 +106,9 @@
 
   // Default TMC slave addresses
   #ifndef E0_SLAVE_ADDRESS
-    #define E0_SLAVE_ADDRESS 0b00
+    #define E0_SLAVE_ADDRESS               0b00
   #endif
+  static_assert(E0_SLAVE_ADDRESS == 0b00, "E0_SLAVE_ADDRESS must be 0b00 for BOARD_BTT_EBB42_V1_1.");
```

Comment:
It is rather odd that some are defined in decimal and others in binary.

Reply:
I don't know why they are that way, but I kept that format to not break anything.